ONSdigital/eq-survey-runner
# Update Travis CI to build master and PRs

### What is the context of this PR?

Currently Travis doesn't build master on PR merges; it should; also as a side-effect this is causing codecov to not report any coverage for master.

The PR changes the .travis.yml to restricts Travis CI to only build pushes to master (i.e. when PRs are merged) but also I have enabled PR building in the Travis CI account settings so that PRs are also built. If you don't restrict Travis to master then it always builds PRs twice, once for the branch that is pushed and once for the PR which is slow and not necessary. 
### How to review
- Check .travis.yml against http://stackoverflow.com/questions/31882306/how-to-configure-travis-ci-to-build-pull-requests-merges-to-master-w-o-redunda
- Verify Travis CI accounts settings for this Repo is set to build PRs and pushes.
- Verify that Travis CI builds this PR once.
- Try pushes a change to the branch and check that Travis re-builds it.
- Verify that on merge to master, the master branch is built by Travis.
### Who worked on the PR
- @collisdigital
